Src/FluentAssertions/Primitives/StringValidatorSupportingNull.cs

```diff
@@ -17,11 +17,6 @@ public StringValidatorSupportingNull(IStringComparisonStrategy comparisonStrateg
 
     public void Validate(string subject, string expected)
     {
-        if (expected is null && subject is null)
-        {
-            return;
-        }
-
```

Comment:
Since we are guarding against `null` in `ContainsEquivalentOf` this path cannot be reached